package.json

```diff
@@ -10,16 +10,15 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@faker-js/faker": "^9.0.0",
-    "bootstrap": "^5.3.3",
-    "husky": "^9.1.6",
     "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-bootstrap": "^2.10.5",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "save": "^2.9.0"
```

Comment:
Whats this used for?

Reply:
react-dom used to escape react if needed(not being used currently) installed it with react bundle at start before started merging react-bootstrap

Reply:
I meant save